Bezier[] pieces
std_msgs/Time generation_time